x-pack/plugins/security_solution/public/timelines/components/timeline/body/actions/header_actions.tsx

```diff
@@ -51,6 +51,24 @@ const SortingColumnsContainer = styled.div`
   }
 `;
 
+const FieldBrowserContainer = styled.div`
+  .euiToolTipAnchor {
```

Comment:
There's a lot going of overwriting of EUI here that I think doesn't need to happen. I know this is part of the actual timeline itself, not the modal, so I think we should cover it in another PR. I'll try to find some time outside of this PR to clean it up a bit.

As just a general rule of thumb, we normally want to write additive selectors rather than do base overwrites. Looking at the code, most of this can be fixed with some flexgroup and button icon props.

Reply:
@snide I agree this is messy. But this was done as a "temporary patch" to show timelines button in security solutions. Later on we'll use the same styles and this will be cleaned.

The rest of the comments are fixed in the last commit.
Thanks!